app/assets/javascripts/admin/spree/spree.js.coffee

```diff
@@ -1,13 +1,11 @@
-#= require jsuri
-
 class window.Spree
   # Helper function to take a URL and add query parameters to it
-  @url: (uri, query) ->
-    if uri.path == undefined
-      uri = new Uri(uri)
-    if query
-      $.each query, (key, value) ->
-        uri.addQueryParam(key, value)
+  @url: (uri) ->
+    console.log(uri)
```

Comment:
Should we keep this `console.log` ?

Reply:
No :see_no_evil: 

Nice catch! :heart: